rules-tests/TypeDeclaration/Rector/Property/TypedPropertyFromAssignsRector/Fixture/skip_not_natively_typed_property_from_data_object.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\TypedPropertyFromAssignsRector\Fixture;
+
+class NotNativelyTypedDataObject
+{
+    /**
+     * @var string
+     */
+    public $notNativelyTypedProperty;
```

Comment:
could you add fixture for has `string` typed property? This class can be moved to `Source` directory

Reply:
Yes, no problem! I will need to have two instances of the both `NotNativelyTypedDataObject` and `NativelyTypedDataObject` (for the `TypedPropertyFromAssignsRector` and `TypedPropertyFromStrictConstructorRector`). Both tests have their own `Source` directory and I can't find a shared one. Shall I just duplicate the classes or do we have another place for them?

Reply:
These directories are already present, but there are two source directories, for both `TypedPropertyFromAssignsRector` and `TypedPropertyFromStrictConstructorRector`. I can do it like so:

![image](https://user-images.githubusercontent.com/50821011/217017612-36d8d983-50ab-4e3f-adc3-814832a00448.png)

But that generates duplicate code. Or I can import the classes from another test, that creates some cross refernces but removes duplication. Adding a `Source` directory in `TypeDeclaration\Rector\Property` seems also strange to me.

Reply:
duplication with different namespace is fine for test 👍

Reply:
Allright, I just pushed that version some minutes ago. It fails on some unrelated files, I think you are already looking into that.  My own changes passes all tests 👌🏻

Reply:
Thank you, please rebase, hopefully it should make it green :)

Reply:
🤞🏻

Reply:
🤞🏻

Reply:
🥳